### ISSUE TITLE
Hang with Term too complex during normalization

### ISSUE THREAD

**Issue by jodavies** [vermaseren/form#323]
It is not yet obvious to me where control goes in the code, but there is a way to obtain the error
```
Term too complex during normalization
... term data ...
Called from Generator
Error in processing one term in thread 3 in module 92
```
and have FORM hang indefinitely, rather than crash out. This is annoying when running large arrays of jobs, for example.

I will keep investigating the logic here, but have just made an issue already to document the problem.

Thanks,
Josh.

**Comment by jodavies** [comment]
Perhaps the call of `Normalize` which doesn't check the return value at `proces.c:3163` is suspicious.

**Comment by jodavies** [comment]
I don't have an easy way to test since I don't have a "minimal example" for this hang, but I wonder if it is related to #14 ?

**Comment by vsht** [comment]
Did you try to compile FORM with the [patch](https://github.com/vermaseren/form/issues/14#issuecomment-639484558) from @tueda ? For me this actually solved all issues with broken TFORM jobs hanging on our cluster so I was about to suggest to add it to the master branch. So far I didn't observe any side effects.

**Comment by jodavies** [comment]
No, but I haven't seen this error since I made this issue, since I changed the way the scripts were doing things to avoid it.

**Comment by jodavies** [comment]
I don’t remember what code I was running here, and don’t recall seeing it since. Let’s close this and revisit if it is ever reported again with a reproducible example.
